website/docs/docs/build/incremental-models.md

```diff
@@ -63,7 +64,7 @@ select
     *,
     my_slow_function(my_column)
 
-from raw_app_data.events
+from {{ ref('app_data_events') }}
```

Comment:
cc @graciegoheen , have added this [here](https://docs-getdbt-p0ueaezu9-dbt-labs.vercel.app/docs/build/incremental-models#filtering-rows-on-an-incremental-run)